website/docs/docs/collaborate/govern/model-contracts.md

```diff
@@ -206,9 +206,13 @@ At the same time, for models with many columns, we understand that this can mean
 When comparing to a previous project state, dbt will look for breaking changes that could impact downstream consumers. If breaking changes are detected, dbt will present a contract error. 
 
 Breaking changes include:
-- Removing an existing column
-- Changing the `data_type` of an existing column
-- Removing or modifying one of the `constraints` on an existing column (dbt v1.6 or higher)
+- Removing an existing column.
+- Changing the `data_type` of an existing column.
+- Removing or modifying one of the `constraints` on an existing column (dbt v1.6 or higher).
+- Removing a contracted model constitutes a breaking change. A model is considered 'removed' if it is deleted, renamed, or disabled:
+  - If the model is versioned, dbt will throw an error.
```

Comment:
Hey @nataliefiann I had spun up a PR for this locally so happen to look at this and suggest these edits:
```suggestion
  - When you remove a versioned model, you see an error.
```